### PR TITLE
Add notes from 2022-06

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: docs/monthly-meeting/2022-06.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Documentation Community Team Meeting (June 2022)
+
+- **Date:** 2022-06-06
+- **Time:** [21:00 UTC](https://arewemeetingyet.com/UTC/2022-04-04/21:00/Docs%20Meeting)
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:4]:
```suggestion
- **Time:** [21:00 UTC](https://arewemeetingyet.com/UTC/2022-06-06/21:00/Docs%20Meeting)
```

File: docs/monthly-meeting/2022-06.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+- Daniele is the author of Diátaxis.
+
+## Reports and celebrations
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- CPython triage permissions for CAM, Pradyun, Adam, and Ned.
+
+
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:41]:
```suggestion
```

File: docs/monthly-meeting/2022-06.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+#### Docs WG renaming to Editorial Board
+
+> If we rename the WG to an Editorial Board, we should clarify the interpretation of the name.
+>
+> A quarterly meeting for the WG seems like the right cadence.
+> 
+> Editorial Board is the agreed name. Wikipedia has a good definition which aligns with our meaning.
+
+Status: There is currently no ceremony/meetings for the work group/editorial board, to meet and make progress on this.
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:68]: Removed "to meet and make progress on this" as I'm not sure anyone suggested the EB having a meeting.
```suggestion
Status: There are currently no meetings for the editorial board.
```

File: docs/monthly-meeting/2022-06.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+- [ ] Petr: Do the rename in all the docs
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:70]:
```suggestion
- [ ] Petr: Do the rename of "working group/WG" to "Editorial Board" in all the docs
```

File: docs/monthly-meeting/2022-06.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+- [ ] Ezio: Add a “what's new” section to the devguide (key/important changes)
+
+CAM: Two different types of content: new pages on tools and workflow versus changes to existing policies and proceduces.
+
+Guido: Not that useful to distinguish these, as it adds more categories. Everything is new.
+
+Ned: Is a problem that we have different audiences? E.g. triagers versus developers versus contributors to Python
+
+Guido: (core devs & triagers), contributors, redistributors/people who need to build Python.
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:124]:
```suggestion
Guido: The three audiences should be (core devs & triagers), contributors, and redistributors/people who need to build Python.
```

File: docs/monthly-meeting/2022-06.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+Ned: What's a workshop?
+
+Daniele: Two 2-hour interactive workshops: theory, interactive exercises, 
+Normally it's in-person.
+
+Ned: Who'd participate?
+
+Daniele: Up to 30 people. Happy to do it for people who wouldn't be working on Python itself. Often do it at conferences.
+
+Adam: should it be people in this call? Core devs?
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:150]:
```suggestion
Adam: Who should be in the workshops? Core devs? This group? Docs contributors?
```

**Review by AA-Turner — COMMENTED** [review]
Minor comments:

A

File: docs/monthly-meeting/2022-06.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+Ezio made some comments in [a thread](https://discord.com/channels/935215565872693329/980818577559007242):
+
+> The first plan is to overhaul a bit the GitHub docs, and is also related to other changes about labels.
+The second plan is still somewhat half-baked, but I wanted to
+> 1. add some instructions about following the development of CPython
+
+Ezio: Add a changelog page to the devguide for an overview of workflow changes
+
+- [ ] Ezio: Add a “what's new” section to the devguide (key/important changes)
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2022-06.md:116]: I created https://github.com/python/devguide/issues/885

File: docs/monthly-meeting/2022-06.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+Ned: Is a problem that we have different audiences? E.g. triagers versus developers versus contributors to Python
+
+Guido: (core devs & triagers), contributors, redistributors/people who need to build Python.
+
+> 2. add a "tips and tricks" page
+> There is some overlapping between https://devguide.python.org/communication/, the instructions that I want to add, the tips and tricks page, and other pages like the GitHub bootcamp though, so I'm still trying to figure out all the things I want to include and then what would be the best way to organize them.
+
+Ned: This sounds like Ezio is suggestion that the devguide should have a 'how to' section (one of the four sections of Diataxis).
+
+- [ ] Ezio: Add a (How-To) Guides section with these
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2022-06.md:131]: I will create an issue/project later.  I still need more time to think what would be the best way to approach this.

File: docs/monthly-meeting/2022-06.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+
+- [ ] Mariatta: to work with Daniele to organize the workshop, CAM & Ned to help with logistics.
+
+#### Workflow
+
+(not discussed due to time)
+
+Should we apply the `skip news` label to documentation PRs? [Discussion in Bedevere's tracker](https://github.com/python/bedevere/issues/457).
+
+Speeding up the `Docs / Docs` CI job -- pros and cons.
```
Comment (AA-Turner) [inline docs/monthly-meeting/2022-06.md:164]: This is now https://github.com/python/core-workflow/issues/459

A

File: docs/monthly-meeting/2022-06.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+
+Ned: Who'd participate?
+
+Daniele: Up to 30 people. Happy to do it for people who wouldn't be working on Python itself. Often do it at conferences.
+
+Adam: should it be people in this call? Core devs?
+
+Daniele: People who are pushing this thru, presenting to others. But skills should be spread far. Need enough people to make it sustainable.
+
+We can probably get more people for a workshop that on this meeting.
```
Comment (hugovk) [inline docs/monthly-meeting/2022-06.md:154]:
```suggestion
We can probably get more people for a workshop than on this meeting.
```

File: docs/monthly-meeting/2022-06.md [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+### Past Action Items
+
+- [ ] Carol (and CAM?) will be updating the readme and guide at the sprints.
+
+Update: CAM didn't see any reviews or feedback on his PRs. (He did take some time off from Python work.)
+
+- [ ] Creating a 'docs-approved' (wording to be agreed) label for CPython.
+
+Needs discussion, please open issue on core-workflow repo.
+(There's a template for label changes: https://github.com/python/core-workflow/issues/new/choose)
```
[the remark this answers is not in the document]
Comment (encukou) [reply docs/monthly-meeting/2022-06.md:53]: I've checked it in Past Action Items in the new agenda.

File: docs/monthly-meeting/2022-06.md (hunk 7 above)
Comment (encukou) [reply docs/monthly-meeting/2022-06.md:116]: I've checked it in Past Action Items in the new agenda.

**Pull request merged**